### PR TITLE
Add auto versioning

### PR DESCRIPTION
**Pull request by Nick-NCSU**

### CODE REVIEW

**Review by JakeFerrero — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -99,15 +99,23 @@ Managed integrations are deployed into the JupiterOne infrastructure by staff
...
-merged into `main`. An administrator of the GitHub project should:
-
-1. Pull the latest from `main`
-1. Determine the new semantic version number
-1. Create the version and tag with `yarn version [--major] [--minor] [--patch]`
-1. Push the commit and tag with `git push --follow-tags`
-
-That's it! Travis will deploy the necessary bits to NPM. Manual deployment is
-possible of course, just be certain to follow the `yarn build` road.
+### Versioning this project
```
Comment (ndowmon) [inline README.md:102]: 🔥

**Review by ndowmon — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -97,8 +97,25 @@
     "@types/lodash.snakecase": "^4.1.6",
     "@types/node": "^13.11.1",
     "@types/uuid": "^8.3.0",
+    "auto": "^10.37.4",
     "dotenv": "^8.2.0",
     "ts-node": "^9.1.1",
     "uuid": "^8.3.2"
+  },
```
Comment (adam-in-ict) [inline package.json:104]: Up in the scripts area, I'd also look at adding a `postversion` command to assist with the use of `auto`.  Something like:

"postversion": "cp package.json ./dist/package.json"

That's a requirement for many projects that package up the dist/ folder for the NPM package.  I'm not as familiar with graph-azure though, so feel free to disregard if not needed.

Comment (adam-in-ict) [inline package.json:104]: Up in the scripts area, I'd also look at adding a `postversion` command to assist with the use of `auto`.  Something like:

"postversion": "cp package.json ./dist/package.json"

That's a requirement for many projects that package up the dist/ folder for the NPM package.  I'm not as familiar with graph-azure though, so feel free to disregard if not needed.

Comment (Nick-NCSU) [reply package.json:104]: Thanks, yeah I ran into that issue when implementing it in `graph-aws`.

**Review by ndowmon — DISMISSED** [review]

**Review by ndowmon — APPROVED** [review]

**Pull request merged**